docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,77 @@
+"""
+Coriolis parameters across the world
```

Comment:
Again, the use of the plural parameters is wrong. It should be parameter. This is clearly confusing, and I therefore assume it could be confusing to users reading this example, so please replace with:

```Deriving the Coriolis frequency over the globe```

This terminology has the possible advantage of being consistent with [Wikipedia](https://en.wikipedia.org/wiki/Coriolis_frequency).